main.cpp

```diff
@@ -367,101 +352,57 @@ void DrawCity(){
 
 
    //gate building 2
-    glBegin(GL_POLYGON);
-    glColor3ub(169, 169, 242);
-    glVertex2i(170,420);
-    glVertex2i(200,420);
-    glVertex2i(200,460);
-    glVertex2i(170,460);
-    glEnd();
+
+    glColor3ub(160, 169, 242);
+    quadHorzInt(170,200,420,460);
 
     //gate_lines
-    glBegin(GL_POLYGON);
-    glColor3ub(0, 0, 0);
-    glVertex2i(177,420);
-    glVertex2i(180,420);
-    glVertex2i(180,460);
-    glVertex2i(177,460);
-    glEnd();
-    glBegin(GL_POLYGON);
-    glColor3ub(0, 0, 0);
-    glVertex2i(190,420);
-    glVertex2i(193,420);
-    glVertex2i(193,460);
-    glVertex2i(190,460);
-    glEnd();
+    glColor3ub(0, 10, 0);
+    quadHorzInt(177,180,420,460);
+
+    glColor3ub(0, 10, 0);
+    quadHorzInt(190,193,420,460);
 
 
 
     // Draw Building One
 
+    //triangle head
     glBegin(GL_POLYGON);
```

Comment:
Please replace this with `triangleNS` function (we have function for triangle too).